packages/contracts/contracts/voting/token/TokenVotingSetup.sol

```diff
@@ -205,7 +204,7 @@ contract TokenVotingSetup is PluginSetup {
             plugin,
             _dao,
             NO_ORACLE,
-            tokenVotingBase.CHANGE_VOTE_SETTINGS_PERMISSION_ID()
+            tokenVotingBase.UPDATE_MAJORITY_VOTING_SETTINGS_PERMISSION_ID()
```

Comment:
```suggestion
            tokenVotingBase.UPDATE_VOTING_SETTINGS_PERMISSION_ID()
```

Reply:
Ammended in c69aa38770f33a15baed838deca116c0c78f7e22